source/form.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+  </head>
+  <body>
+    <nav class="main-navigation">
+      <ul class="main-menu">
+        <li class="main-menu__item">
+          <a class="main-menu__link" href="index.html">Главная</a>
+          <button class="main-menu__button main-menu__button_hide-menu" type="button"></button>
+        </li>
+        <li class="main-menu__item">
+          <a class="main-menu__link" href="photo.html">Фото и видео</a>
+        </li>
+        <li class="main-menu__item">
+          <a class="main-menu__link" href="form.html">Форма отзыва</a>
+        </li>
+        <li class="main-menu__item">
+          <a class="main-menu__link" href="https://htmlacademy.ru/intensive/adaptive">Html Academy</a>
+        </li>
+      </ul>
+    </nav>
+    <header class="page-header">
+      <img class="sedona-logo" src="https://via.placeholder.com/150x150" alt="Логотип Sedona">
+      <button class="main-header__button main-header__button_open-menu" type="button"></button>
+    </header>
+    <main class="page-main">
+      <h1 class="feedback__heading">Оставьте свой отзыв</h1>
+      <section class="simple-greeting">
+        <img class="simple-greeting__img" src="https://via.placeholder.com/150x150" alt="">
+      </section>
+      <section class="page-content">
+        <div class="feedback">
+          <h2 class="feedback__heading">Оставьте свой отзыв</h2>
+            <p class="feedback__text">Помогите нашим отелям стать лучше! Оставьте отзыв оних, а также о посещенных вами достопримечательностях</p>
+        </div>
+        <form action="https://echo.htmlacademy.ru" method="post">
```

Comment:
Почему нет класса на форме?
```suggestion
        <form action="https://echo.htmlacademy.ru" class="feedback-form" method="post">
```